entrypoint/50-letsencrypt-renew.sh

```diff
@@ -1,17 +1,7 @@
 #!/bin/sh
-set -eu
```

Comment:
please keep `set -u` there is no reason to remove it

Reply:
I removed that one cause there are no variables being substituted here. I will bring it back cause I guess it's important for consistency across projects